### PR TITLE
edit the readme file

### PR DESCRIPTION
**Pull request by r-lehmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,106 @@
...
 
-These steps should be completed before you get started:
+### Prerequisites
 
-1. You need a **Kameleoon Account**.
+Make sure you have the following requirements before you get started:
 
-2. You need an AWS Account with Lambda@Edge Access. For more information, visit the official [AWS Lambda@Edge product page here](https://aws.amazon.com/lambda/edge).
+1. A Kameleoon user account. Visit [kameleoon.com](https://www.kameleoon.com/) to learn more.
+1. The [Kameleoon NodeJS SDK](https://developers.kameleoon.com/feature-management-and-experimentation/web-sdks/js-sdk) installed with some feature flags or experiments already configured.
```
Comment (esenzh) [inline README.md]: incorrect link, it redirects to JS SDK instead of NodeJS SDK

File: README.md [hunk 2]
```diff
@@ -1,101 +1,106 @@
...
+You'll use these files to initialize your environment:
 
-> **Lambda Layers**: If you need additional libraries, custom runtimes, or configuration files to use alongside your Lambda function, consider looking into utilizing [Lambda Layers](https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html).
+1. Navigate to `src/viewer_request.js` and update the `YOUR_SITE_CODE` and `YOUR_FEATURE_KEY` values with your [Kameleoon credentials](https://help.kameleoon.com/api-credentials).
+1. Review or adjust the feature flags or experiments you've configured with the Kameleoon NodeJS SDK, and hook into the lifecycle events by inserting your desired logic into the switch-case statements in `src/viewer_request.js` and `src/viewer_response.js`.  For example, you can change headers, cookies, and more.
+1. Run `npm run build:viewer_request` to use Rollup to bundle the source code of `src/viewer_request.js` into a `dist/dist.zip` file that you'll import into Lambda. The `dist.zip`` file should be roughly **~49kb** in size for `src/viewer_request.js` assuming you have not made any additional changes.
```
Comment (esenzh) [inline README.md]: `dist.zip`` => remove extra back tick

File: README.md [hunk 3]
```diff
@@ -1,101 +1,106 @@
...
+1. Upload the request `dist/dist.zip` file into Lambda using one of these options:
+    -**GUI**: Go to your AWS Lambda console, select the function associated with your Lambda environment, and import the `dist.zip` file. After you upload it, there should now be a minified `index.js` file located inside of your Lambda function's **Code Source** section.
+    **CLI**: You can use the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to update your AWS Lambda function programmatically. Example command: `aws lambda update-function-code --function-name my-aws-lambda-at-edge-function --zip-file fileb://dist.zip`.
```
Comment (esenzh) [inline README.md]: these two GUI and CLI are not starting from new line in view mode

File: README.md (hunk 1 above)
Comment (r-lehmann) [reply README.md]: Good catch! Thanks!

File: README.md (hunk 3 above)
Comment (r-lehmann) [reply README.md]: should be fixed now.

File: README.md (hunk 2 above)
Comment (r-lehmann) [reply README.md]: fixed.

**Pull request merged**